tests/unit/countries-provinces-mixin.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { shallowMount } from '@vue/test-utils'
+import MixinTester from '@/mixin-tester.vue'
+import { nextTick } from 'vue'
+
+const canadaProvinces = [
+  { name: 'Alberta', short: 'AB' },
+  { name: 'British Columbia', short: 'BC' },
+  { name: 'Manitoba', short: 'MB' },
+  { name: 'New Brunswick', short: 'NB' },
+  { name: 'Newfoundland and Labrador', short: 'NL' },
+  { name: 'Northwest Territories', short: 'NT' },
+  { name: 'Nova Scotia', short: 'NS' },
+  { name: 'Nunavut', short: 'NU' },
+  { name: 'Ontario', short: 'ON' },
+  { name: 'Prince Edward Island', short: 'PE' },
+  { name: 'Quebec', short: 'QC' },
+  { name: 'Saskatchewan', short: 'SK' },
+  { name: 'Yukon', short: 'YT' }
+]
+
+const canadaProvincesExcludeBC = [
+  { name: 'Alberta', short: 'AB' },
+  { name: 'Manitoba', short: 'MB' },
+  { name: 'New Brunswick', short: 'NB' },
+  { name: 'Newfoundland and Labrador', short: 'NL' },
+  { name: 'Northwest Territories', short: 'NT' },
+  { name: 'Nova Scotia', short: 'NS' },
+  { name: 'Nunavut', short: 'NU' },
+  { name: 'Ontario', short: 'ON' },
+  { name: 'Prince Edward Island', short: 'PE' },
+  { name: 'Quebec', short: 'QC' },
+  { name: 'Saskatchewan', short: 'SK' },
+  { name: 'Yukon', short: 'YT' }
+]
```

Comment:
If you want to play, you could save yourself some duplication here by either:
- create this array as a copy of the previous array but filter out BC
- create the other array as a copy of this array but add in BC
😃

Reply:
Fixed